lib/luadata.c

```diff
@@ -13,9 +13,12 @@
 
 #include <lunatik.h>
 
+#include "luadata.h"
+
 typedef struct luadata_s {
 	char *ptr;
 	size_t size;
+	uint32_t flags;
```

Comment:
if you are using `typedef enum ... luadata_flags_t`, you should use this type here, right?